### ISSUE TITLE
E3002 Invalid Property Resources/TaskDefinition/Properties/Volumes/0/efsVolumeConfiguration

### ISSUE THREAD

**Issue by f0o** [aws-cloudformation/cfn-python-lint#1638]
*cfn-lint version: (`cfn-lint --version`)*
cfn-lint 0.34.1

*Description of issue.*
Newly added EFS VolumeConfiguration for ECS TaskDefinitions not yet supported

Cfn support: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-taskdefinition-volumes.html

`cfn-lint -u` does not rectify the issue, perhaps the definitions are not yet updated upstream?

**Comment by PatMyron** [comment]
[Looks like this property got picked up in most of the regions but not the default `us-east-1` yet](https://github.com/ScriptAutomate/aws-cfn-resource-specs/blob/master/CHANGELOG.md#existing-resourcetypes-and-propertytypes-not-in-us-east-1)

[Inline resource-level ignores](https://github.com/aws-cloudformation/cfn-python-lint/#resource-based-metadata) should mitigate until then:
```yaml
  Resource:
    Type: AWS::ECS::TaskDefinition
    Metadata:
      cfn-lint:
        config:
          ignore_checks:
          - E3002
```

more general tracking issue: https://github.com/aws-cloudformation/cfn-python-lint/issues/1219

**Comment by PatMyron** [comment]
[now available in the latest `us-east-1` Resource Specification](https://d1uauaxba7bl26.cloudfront.net/latest/gzip/CloudFormationResourceSpecification.json)

`cfn-lint -u` / `cfn-lint --update-specs` should now pick this up and this should be included in next release

**Comment by jedis00** [comment]
This appears to still be broken. I updated cfn-lint to 0.35.0 and it still does not recognize the property:

```cfnlint: error
3002 - Invalid Property Resources/ConsumerTaskDefinition/Properties/Volumes/0/EFSVolumeConfiguration```

**Comment by kddejong** [comment]
@jedis00 this template is not returning an error for me.  Did you also try the `cfn-lint --update-specs`?  
```
Resources:
  taskdefinition: 
    Type: AWS::ECS::TaskDefinition
    Properties: 
      Volumes: 
        - EFSVolumeConfiguration:
            FilesystemId: "id"
```

**Comment by jedis00** [comment]
@kddejong I figured it out. I tried the update-specs first, and it didn't work. Then updated cfn-lint, and still no dice. Had to restart Sublime and then it started working. Not sure if both were needed, but it doesn't hurt. Thanks!
